### ISSUE TITLE
complains about alternate `[dependency]` blocks showing a git include

### ISSUE THREAD

**Issue by jmesmon** [mgeisler/version-sync#42]
For example, https://github.com/jmesmon/rust-systemd/blob/cec9df440144daf975348fb7532f88c4757ca9ec/README.md or https://github.com/jmesmon/rust-systemd/blob/bb87c893127da33cf27c32fcbfd40a40d5f9cb37/README.md

```toml
[dependencies.systemd]
git = "https://github.com/jmesmon/rust-systemd"
```

etc.

Marking as `no_sync` as a workaround, but seems plausible that `version-sync` could detect that one is trying to provide a git dependency and allow it.

**Comment by mgeisler** [comment]
Hi Cody,

Thanks for the report -- I agree completely, `version-sync` should be smart enough to ignore such dependencies. I'll see what I can come up with!

**Comment by ahmedcharles** [comment]
When might this be available on crates.io?

**Comment by mgeisler** [comment]
Hi @ahmedcharles, I just merged #45 to address this issue. I'll try to make a release in the next days.

**Comment by ahmedcharles** [comment]
Thanks!
